### PR TITLE
updated the BRS for storage TLS  and docs for TBv8 control

### PR DESCRIPTION
**Pull request by pranchalsomani**

### CODE REVIEW

File: Control coverage/Feature/DBForMySqlFlexibleServer.md [hunk 1]
```diff
@@ -4,15 +4,15 @@
...
 <!-- /TOC -->
 <br/>
 
 ___ 
 
-## Azure_DBForMySQLFlexibleServer_DP_Enable_SSL_Trial 
+## Azure_DBForMySQLFlexibleServer_DP_Enable_SSL
 
 ### Display Name 
 [Trial] Azure Database for MySQL - Flexible Servers Announcing SSL enforcement
```
Comment (asimks) [inline Control coverage/Feature/DBForMySqlFlexibleServer.md:18]: Trail should be removed from display name too

File: Scripts/RemediationScripts/Remediate-SetStorageAccountMinReqTLSVersion.ps1 [hunk 2]
```diff
@@ -1,39 +1,39 @@
...
-    This script is used to set required TLS version for Storage Account in a Subscription.
+    This script is used to set required minimum TLS version for Storage Accounts in a Subscription.
 
 # Control ID:
-    Azure_Storage_DP_Use_Secure_TLS_Version_Trial
+    Azure_Storage_DP_Use_Secure_TLS_Version
 
 # Display Name:
-    Use Approved TLS Version in Storage Account.
+    Use Approved minimum TLS version in Storage Account.
```
Comment (asimks) [inline Scripts/RemediationScripts/Remediate-SetStorageAccountMinReqTLSVersion.ps1:9]: Keep it same as one present in control json file.

File: Scripts/RemediationScripts/Remediate-SetStorageAccountMinReqTLSVersion.ps1 [hunk 3]
```diff
@@ -1,39 +1,39 @@
...
 
 # Prerequisites:
     1. Contributor or higher privileges on the Storage Accounts in a Subscription.
     2. Must be connected to Azure with an authenticated account.
 
 # Steps performed by the script:
     To remediate:
-        1. Validate and install the modules required to run the script.
-        2. Get the list of Storage Accounts in a Subscription that do not use the required TLS version
+        1. Validate and install the modules required to run the script and validating the user..
```
Comment (asimks) [inline Scripts/RemediationScripts/Remediate-SetStorageAccountMinReqTLSVersion.ps1:17]: Validate and install the modules required to run the script and validating the user.. -> Validate and install the modules required to run the script and **validate the user.**

File: Scripts/RemediationScripts/Remediate-SetStorageAccountMinReqTLSVersion.ps1 [hunk 4]
```diff
@@ -1,39 +1,39 @@
...
+        2. Get the list of Storage Accounts in a Subscription that do not use the required minimum TLS version.
         3. Back up details of Storage Accounts that are to be remediated.
-        4. Set the required TLS version on the all Storage Accounts in the Subscription.
+        4. Set the required minimum TLS version on the all Storage Accounts in the Subscription.
 
     To roll back:
-        1. Validate and install the modules required to run the script.
+        1. Validate and install the modules required to run the script and validating the user..
```
Comment (asimks) [inline Scripts/RemediationScripts/Remediate-SetStorageAccountMinReqTLSVersion.ps1:23]: validate the user.
Remove extra full stop

File: Scripts/RemediationScripts/Remediate-SetStorageAccountMinReqTLSVersion.ps1 [hunk 5]
```diff
@@ -261,7 +261,7 @@ function Set-StorageAccountRequiredTLSVersion
     Write-Host "To Set Minimum TLS version for Storage Accounts in a Subscription, Contributor or higher privileges on the Storage Accounts are required." -ForegroundColor $([Constants]::MessageType.Warning)
     Write-Host $([Constants]::SingleDashLine)
 
-    Write-Host "[Step 2 of 5] Fetch all Storage Accounts"
+    Write-Host "[Step 2 of 4] Fetching Storage Accounts with Minimum TLS less than required minimum TLS version..."
```
Comment (asimks) [inline Scripts/RemediationScripts/Remediate-SetStorageAccountMinReqTLSVersion.ps1:264]: Fetching Storage Accounts with Minimum TLS less than required minimum TLS version... -> Fetching Storage Accounts with Minimum TLS **version** less than required minimum TLS version...

File: Scripts/RemediationScripts/Remediate-SetStorageAccountMinReqTLSVersion.ps1 [hunk 6]
```diff
@@ -421,7 +418,7 @@ function Set-StorageAccountRequiredTLSVersion
      
     if ($totalStorageAccountsWithoutReqMinTLSVersion  -eq 0)
     {
-        Write-Host "No Storage Account(s) found where TLS is less than required TLS version.. Exiting..." -ForegroundColor $([Constants]::MessageType.Update)
+        Write-Host "No Storage Account(s) found where minimum TLS is less than required minimum TLS version.. Exiting..." -ForegroundColor $([Constants]::MessageType.Update)
```
Comment (asimks) [inline Scripts/RemediationScripts/Remediate-SetStorageAccountMinReqTLSVersion.ps1:421]: minimum TLS is less -> minimum TLS **version** is less

File: Control coverage/Feature/DBForMySqlFlexibleServer.md [hunk 7]
```diff
@@ -33,17 +33,17 @@ Enforcing secure transport between your database server and your client applicat
...
 - **Azure Portal** 
-To configure secure transport for client communication, Go to Azure Portal --> Azure Database for MySQL flexible server -->Server parameters --> set parameter 'require-secure-transport' as 'ON'.
+To configure secure transport for client communication, Go to Azure Portal --> Azure Database for MySQL flexible server --> Select server --> Settings --> Server parameters --> search 'require_secure_transport' --> set parameter 'require_secure_transport' as 'ON' --> Click 'Save'.
 
-<!-- - **PowerShell** 
+- **PowerShell** 
 
 	 ```powershell 
-	 $variable = 'apple' 
+	 Update-AzMySqlFlexibleServerConfiguration -Name "require_secure_transport"  -ResourceGroupName <ResourceGroupName>  -ServerName <ServerName> -Value "ON"
```
Comment (jaiswalricha) [inline Control coverage/Feature/DBForMySqlFlexibleServer.md:46]: does this script not provide the remediation on subscription level?

File: Control coverage/Feature/Storage.md [hunk 8]
```diff
@@ -223,13 +223,13 @@ TLS provides privacy and data integrity between client and server. Using approve
 > Any of the following conditions is met.
 > * Minimum TLS version is not set (default 1.0).
```
Comment (jaiswalricha) [inline Control coverage/Feature/Storage.md:224]: better to write if min TLS version is less than 1.2 (configured min required tls version)

File: Scripts/RemediationScripts/Remediate-SetStorageAccountMinReqTLSVersion.ps1 (hunk 5 above)
Comment (jaiswalricha) [reply Scripts/RemediationScripts/Remediate-SetStorageAccountMinReqTLSVersion.ps1:264]: Fetch Storage Accounts with Minimum TLS version less than required minimum TLS version

File: Scripts/RemediationScripts/Remediate-SetStorageAccountMinReqTLSVersion.ps1 [hunk 9]
```diff
@@ -440,16 +437,16 @@ function Set-StorageAccountRequiredTLSVersion
         return
     }
 
-    Write-Host "Found [$($totalStorageAccountsWithoutReqMinTLSVersion)] Storage Accounts where TLS version is either not set or less than required Minimum TLS version." -ForegroundColor $([Constants]::MessageType.Update)
+    Write-Host "Found [$($totalStorageAccountsWithoutReqMinTLSVersion)] Storage Accounts where minimum TLS version is either not set or less than required Minimum TLS version." -ForegroundColor $([Constants]::MessageType.Update)
     Write-Host $([Constants]::SingleDashLine)	
      
     if(-not($AutoRemediation))
     {
-        Write-Host "Following Storage Accounts are having TLS version either not set or less than required Minimum TLS version less than required Minimum TLS Version:" -ForegroundColor $([Constants]::MessageType.Info)
```
Comment (jaiswalricha) [inline Scripts/RemediationScripts/Remediate-SetStorageAccountMinReqTLSVersion.ps1:448]: Minimum TLS version (make all version either start with V or v, consistency missing)

File: Scripts/RemediationScripts/Remediate-SetStorageAccountMinReqTLSVersion.ps1 [hunk 10]
```diff
@@ -464,7 +461,7 @@ function Set-StorageAccountRequiredTLSVersion
     }
 
     
-    Write-Host "[Step 4 of 5] Backing up Storage Account(s) details"
+    Write-Host "[Step 3 of 4] Backing up Storage Account(s) details..."
```
Comment (jaiswalricha) [inline Scripts/RemediationScripts/Remediate-SetStorageAccountMinReqTLSVersion.ps1:464]: back up storage

File: Scripts/RemediationScripts/Remediate-SetStorageAccountMinReqTLSVersion.ps1 [hunk 11]
```diff
@@ -787,9 +758,9 @@ function Reset-StorageAccountRequiredTLSVersion
 
     # Note about the required access required for remediation
 
-    Write-Host "To reset TLS Versions for Storage Account(s) in a Subscription, Contributor or higher privileges on the Storage Account(s) are required." -ForegroundColor $([Constants]::MessageType.Info)
```
Comment (jaiswalricha) [inline Scripts/RemediationScripts/Remediate-SetStorageAccountMinReqTLSVersion.ps1:790]: to reset the minimum TLS version for

**Review by asimks — APPROVED** [review]

**Review by jaiswalricha — APPROVED** [review]

**Pull request merged**